### PR TITLE
Fix: APP-2563 - Delegation Cleanup

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description

- clears errors when Delegation Modal is closed
- increases max height of the Modal
- updates Delegation Modal text styling
- removes Community `AlertInline`
- fixes infinite rendering issue on Community page
- fixes mobile pagination for total pages less than 3.

Task: [APP-2563](https://aragonassociation.atlassian.net/browse/APP-2563)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2563]: https://aragonassociation.atlassian.net/browse/APP-2563?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/community.tsx [hunk 1]
```diff
@@ -271,9 +270,6 @@ export const Community: React.FC = () => {
                 />
               )}
             </div>
-            {!walletBased && (
```
Comment (cgero-eth) [inline src/pages/community.tsx:274]: We can probably remove the wrapping div with `className="space-y-4"` as well, right?

**Review by cgero-eth — COMMENTED** [review]
Thanks for the fixes @Fabricevladimir 👍🏻

**Comment by Fabricevladimir** [comment]
@RuggeroCino Please test with this [DAO](https://staging-app.aragon.org/#/daos/ethereum/falconia.dao.eth/community). There seems to have been an infinite loop and clicking on the page number buttons didn't trigger the pagination.

**Comment by cgero-eth** [comment]
> @RuggeroCino Please test with this [DAO](https://staging-app.aragon.org/#/daos/ethereum/falconia.dao.eth/community). There seems to have been an infinite loop and clicking on the page number buttons didn't trigger the pagination.

Yeah weird, looks like it is fixed now well done 👍🏻

**Review by cgero-eth — APPROVED** [review]

**Pull request merged**